src/System.Device.Gpio/Interop/Unix/libgpiod/V1/SafeChipHandle.cs

```diff
@@ -1,9 +1,13 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+// Disable these StyleCop rules for this file, as we are using native names here.
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+
 using System.Runtime.InteropServices;
+using LibgpiodV1 = Interop.LibgpiodV1;
 
-namespace System.Device.Gpio;
+namespace System.Device.Gpio.Interop.Unix.libgpiod.V1;
```

Comment:
I understand your logic here, but 7 levels of namespace hirerarchy might be pushing it (even when only having non-public types), so I'd suggest to stay with something like `System.Device.Gpio.Interop.GpiodV1` or something along those lines.

Reply:
Didn't know your convention so I went with file location. Will change it to your suggestion